bootstrapping_decision_making.md

```diff
@@ -1,34 +1,34 @@
-# Bootstrapping Decision-Making Bodies
+# Bootstrapping Subproject Councils
```

Comment:
How about Project Management Council?

Reply:
FWIW, in the new governance model the "Project" is Jupyter and the different software "things" are Subprojects.